frontend/packages/dev-console/integration-tests/features/operators.feature

```diff
@@ -93,6 +93,16 @@ Scenario: Create HyperConverged Cluster: VM-01-TC02
     Then user will see a HyperConverged Cluster created
     And user will see Virtualization item under Workloads
 
+@regression @smoke
+Scenario: Quay container security operator
+   Given user is at Operator Hub page with the header name "OperatorHub"
+   When user searches for "quay container security"
+   And user clicks on quay container security operator card on Operator Hub page
+   And user clicks install button present on the right sidebar
+   And user installs the quay container security operator with default values
```

Comment:
This is the scenario of installing the operator. Rather than this statement, can you please update this statement in such a way that it defines the action and not the whole process? 
For example, user clicks on install button

Reply:
Same as above